### ISSUE TITLE
Feedback on the new Anthology website

### ISSUE THREAD

**Issue by mbollmann** [acl-org/acl-anthology#170]
This thread is intended to collect all feedback, suggestions, bug reports, etc. for the new Anthology website in the [`static-rewrite`](https://github.com/acl-org/acl-anthology/tree/static-rewrite) branch.

(Edit: [live demo here at http://aclweb.org/anthology](http://aclweb.org/anthology))

**If you do not have a GitHub account, you're also welcome to send me feedback via e-mail ([marcel@bollmann.me](mailto:marcel@bollmann.me)) or Twitter ([\@mmbollmann](https://twitter.com/mmbollmann))!**

## Known Issues

+ The **search functionality** now uses Google Custom Search. We're still finetuning its settings and waiting for some pages to be indexed, so please don't report any weird search behaviour just yet.
+ **Author name variations** (#86) are an open problem that we plan to address before the site launch.

**Comment by akoehn** [comment]
I really like it, especially the speed!

There is a `display:none` span containing the text "bib" in the bibtex block inside the `acl-paper-link-block` block. When using a text browser, this leads to the text being BibTeXbib.  That span should be removed.

As a minor comment: Could you specify the hardware requirements for building the anthology a bit? How much time & memory does building take? "a considerable amount of memory" could be 8GB or 512, depending on whom you ask :-)

**Comment by davidweichiang** [comment]
It looks great! On Safari, when you click on on pdf/bib link and then click the browser's back button, the little callout ("Open PDF" or "Export BibTeX") remains on.

**Comment by danielgildea** [comment]
awesome!!!!!!!!!

**Comment by desilinguist** [comment]
Looks awesome! Great work! 👏

**Comment by mjpost** [comment]
What's the reason for inserting newlines in the bib field values? (for example, [in booktitle here](http://www.aclweb.org/anthology/papers/N/N18/N18-1120.bib), and titles elsewhere).

**Comment by stevenbedrick** [comment]
_Disclaimer_: This is about search, but is not about weird search behavior as such. Is Google Custom Search the long-term search solution for the new version of the Anthology? It is inherently waaaaay less functional than the existing search system on the current Anthology- for example, the current search page has really great result faceting, etc.

**Comment by stevenbedrick** [comment]
And I just saw #165 - glad to see that something more flexible is on the roadmap/radar. In the meantime, we could also link to the DFKI "[ACL Anthology Searchbench](http://aclasb.dfki.de)".

**Comment by aryamccarthy** [comment]
On mobile, the magnifying glass of the search bar gets forced to the next row for me.

**Comment by aryamccarthy** [comment]
Is the BibTeX generation handling special characters properly? 

This entry has weird quotation marks in the abstract. <http://aclweb.org/anthology/papers/C/C18/C18-1137.bib>
This one has weird things going on in the title field. <http://aclweb.org/anthology/papers/K/K18/K18-3001.bib>

**Comment by danielhers** [comment]
When there is just one paper in a conference, the noun after the number should be singular "paper" and not "papers".
Example:  ```Proceedings of the Pilot SENSEVAL 1 papers``` in http://www.aclweb.org/anthology/venues/semeval/

**Comment by rahular** [comment]
Awesome work! One small issue I saw is that when I am browsing through papers in pages like [this](http://aclweb.org/anthology/events/acl-2018), there is no way for me to scroll back to the top instantly. The `up` button which is present at the beginning of the page could be floating around a corner.

**Comment by danielgildea** [comment]
> Is the BibTeX generation handling special characters properly?

Fixed by [6bbc5a1f4f35744f609f384e866a95bf6cc8f021]

**Comment by davidweichiang** [comment]
Re: https://github.com/acl-org/acl-anthology/issues/170#issuecomment-471835229, when I view in Chrome or iOS Safari, I see mojibake, but on macOS Safari, it looks fine.

Although @danielgildea's fix puts the .bib file into ASCII (as it should be), I wonder if, as a failsafe, can the server put `Content-Type: application/x-bibtex; charset=utf-8` into the response header?

**Comment by danielgildea** [comment]
> What's the reason for inserting newlines in the bib field values? (for example, [in booktitle here](http://www.aclweb.org/anthology/papers/N/N18/N18-1120.bib), and titles elsewhere).

`anth2bib.py` is just passing through newlines that are in the titles in the xml files.  
I can't figure out where they come from originally.  Personally, I think they make the bibtex more readable anyway.

`anth2bib.py` does insert newlines between author names.  I think this makes it more readable, 
especially when names are in "Last, First" format.

**Comment by aryamccarthy** [comment]
> > Is the BibTeX generation handling special characters properly?
> 
> Fixed by [[6bbc5a1](https://github.com/acl-org/acl-anthology/commit/6bbc5a1f4f35744f609f384e866a95bf6cc8f021)]

I'm seeing "CoNLLâ€“SIGMORPHON" in macOS Safari, instead of "CoNLL–SIGMORPHON". Does the build script need to be re-run to show the fix?

**Comment by mbollmann** [comment]
> Does the build script need to be re-run to show the fix?

Absolutely. Fixes are not reflected on the live website until @mjpost rebuilds it and pushes it there.

**Comment by mjpost** [comment]
I agree the one-line-per-author variant is more readable and is fine with me, as long as we make sure to use spaces and not tabs (per #16).

I'll rebuild soon, by tonight at the latest. Once we have continuous integration checks built (#102) and other checks against commits to the master branch, we can have it automated.

**Comment by mbollmann** [comment]
Thanks for all the feedback so far! I've implemented a bunch of minor layout fixed based on the comments here (with the same caveat as above: will not be live until Matt rebuilds).

> _Disclaimer_: This is about search, but is not about weird search behavior as such. Is Google Custom Search the long-term search solution for the new version of the Anthology? It is inherently waaaaay less functional than the existing search system on the current Anthology- for example, the current search page has really great result faceting, etc.

I believe Google Custom Search is much more powerful than people give it credit for, and it offers customization options that should allow for similar result faceting and features as before. However, that requires some more work on my part, and it wasn't really possible to implement and test this earlier as, by its very nature, it requires the new site to be live and getting indexed by Google first.

I'd really like to advocate for some more patience here over the coming weeks as I'm hoping to improve this. Maintaining a custom-made search solution is a huge liability IMO, and I would really like for people to give the Google version a fair chance first.

**Comment by stevenbedrick** [comment]
@mbollmann That's totally fair, and thank you for the reply. I certainly see the value of using an off-the-shelf/hosted search platform in general, and also of using Google Custom Search in particular as a "getting things up and running" solution. For the sake of clarity, my concerns are less about the _search_ behavior of GCS- if anybody can build a decent text search engine, it'd be Google! My concerns are more about search UI/UX- result faceting, etc. I'm happy to give GCS more of a chance, and am looking forward to seeing what we're able to do with GCS in terms of customization. Thank you (all of you!) for your efforts on this project; I do very much like the redesign overall and am excited to see it evolve!

**Comment by mjpost** [comment]
Okay, rebuilt. I also merged in master which had some corrections.

**Comment by aryamccarthy** [comment]
Unclear whether this is a parsing error or a data error: [this BibTeX](http://aclweb.org/anthology/papers/D/D13/D13-1088.bib) has no article title.

**Comment by mjpost** [comment]
Thanks! The title appears in the HTML: (http://aclweb.org/anthology/D13-1088/) and is [in the XML](https://github.com/acl-org/acl-anthology/blob/4a751ac51c47d6636f0b682fa8a3816c8167a2cd/import/D13.xml#L1438), so I'm not sure what's going on here.

**Comment by mbollmann** [comment]
> Thanks! The title appears in the HTML: (http://aclweb.org/anthology/D13-1088/) and is [in the XML](https://github.com/acl-org/acl-anthology/blob/4a751ac51c47d6636f0b682fa8a3816c8167a2cd/import/D13.xml#L1438), so I'm not sure what's going on here.

Pretty sure it's related somehow to the title starting with `<fixed-case>`. It's fixed with the refactored BibTeX generation in 7cd20c3.

**Comment by mjpost** [comment]
Ah, I was looking at the master branch. i’ll rebuild tonight.

**Comment by mjpost** [comment]
Done, and the problem [is indeed fixed](http://aclweb.org/anthology/papers/D/D13/D13-1088.bib). Thanks!

**Comment by aryamccarthy** [comment]
Another question of is-it-the-data-or-the-site:

SIGMORPHON has workshops listed [through 2014](http://aclweb.org/anthology/sigs/sigmorphon/), but one of the 2014 ones is really 2016: W16-20. On top, [their 2018 workshop](http://aclweb.org/anthology/events/ws-2018/#w18-58) isn't listed on their page.

**Comment by mbollmann** [comment]
> Another question of is-it-the-data-or-the-site:
> 
> SIGMORPHON has workshops listed [through 2014](http://aclweb.org/anthology/sigs/sigmorphon/), but one of the 2014 ones is really 2016: W16-20. On top, [their 2018 workshop](http://aclweb.org/anthology/events/ws-2018/#w18-58) isn't listed on their page.

It's the data:
https://github.com/acl-org/acl-anthology/blob/static-rewrite/import/sigmorphon.yaml

The W16-20 one is tagged as 2014 there, and there's no entry for 2018. You can submit a PR or I can fix it sometime later.

**Comment by mjpost** [comment]
@aryamccarthy, could you submit a PR (against the `static-rewrite` branch)?

**Comment by nschneid** [comment]
Clarification question: Which will be the permanent home of the anthology, aclweb.org/anthology or aclanthology.info? Re: https://github.com/zotero/translators/issues/1702#issuecomment-475880213

**Comment by mbollmann** [comment]
> Clarification question: Which will be the permanent home of the anthology, aclweb.org/anthology or aclanthology.info? Re: [zotero/translators#1702 (comment)](https://github.com/zotero/translators/issues/1702#issuecomment-475880213)

The former (aclweb.org/anthology). There's already an issue (#178) to add 301 redirects from the aclanthology.info site, they're just not up yet.

**Comment by Evpok** [comment]
Quick question (I'll open a dedicated feature request if people are interested): is there any hope of supporting BibLaTeX exports? The data model of the default styles include keys like `eventdate`, `eventtitle` and `eventtitleaddon` that could be used for the confs' date, full name and short name repectively, `booksubtitle` that would allow to remove the volume title from the proceedings title and so on. That would make for cleaner metadata.

**Comment by mjpost** [comment]
Would you need individual files for every paper, or would it be sufficient to dump a single BibLaTeX file for the entire anthology? We current do both for BibTex (you can get [the whole archive here [5 MB]](http://www.aclweb.org/anthology/anthology.bib.gz)).

**Comment by Evpok** [comment]
Ideally for every paper for feature parity with the BibTeX version, but I understand it could be impractical, so a single one for the whole anthology would be a good start :-) (at least for the next 10 years until me and other proselytes manage to kill BibTeX for good).
I can volunteer for doing the work by the way, whatever solution you chose

**Comment by BshoterJ** [comment]
awesome!

**Comment by wuzhen247** [comment]
cool

**Comment by GabrielLin** [comment]
I argue you to get back the EndNote importer file. Thanks.

**Comment by mjpost** [comment]
Hi @GabrielLin,

Thanks for the feedback. We have created a new issue to track this (#235), and have a question for you there.

**Comment by seeledu** [comment]
I‘m a Chinese user, I can't use the Search Results.

**Comment by seeledu** [comment]
> I‘m a Chinese user, I can't use the Search Results.

and when i use the VPN, it works well.

**Comment by akoehn** [comment]
@seeledu, does google.com work without VPN? If not, that's the problem :-/

**Comment by knmnyn** [comment]
Google products are blocked by the Great Firewall of China occasionally (more often than not).  This is why relying on G products for international services that have a large membership in CN (i.e., ACL) is not usually a good idea.  

As ACL is an international organization, we may want to think more about this issue (and maybe get the official word from the ACL Exec).  Previously the Rails search allowed a pretty comprehensive search within the system, but now perhaps searching the static site would be easier in some ways.

**Comment by Franck-Dernoncourt** [comment]
On the author page, e.g. https://www.aclweb.org/anthology/people/y/yan-song/, we can only see the count for the top 5 most frequent venues (it used to be possible to view the count for all venues):

![image](https://user-images.githubusercontent.com/15331/55843178-c7f44680-5aeb-11e9-8290-f84a52ee35af.png)

**Comment by yucc2018** [comment]
papers(PDF full) in the following pages can't be downloaded.
https://www.aclweb.org/anthology/events/cl-2018/

**Comment by malikalamgirian** [comment]
Where can I find the appendices? I could not find the appendices to the papers anywhere for EMNLP 2018.

**Comment by mbollmann** [comment]
> Where can I find the appendices? I could not find the appendices to the papers anywhere for EMNLP 2018.

Supplementary material can be accessed via the green buttons. For EMNLP 2018, they all seem to be labeled "Attachment". If that doesn't answer your question, can you clarify?

**Comment by malikalamgirian** [comment]
I still could not find the attachments for this paper https://aclweb.org/anthology/papers/D/D18/D18-1514/

It mentions on page 4805 that Appendix A.2 contains some related materials, but I still could not find the Appendix or the attachments anywhere.

**Comment by mbollmann** [comment]
> I still could not find the attachments for this paper https://aclweb.org/anthology/papers/D/D18/D18-1514/

There don't seem to be any additional files for this submission on the server, so my first guess would be that despite what they write in the paper, the authors didn't actually provide an appendix. Maybe someone involved in ingesting EMNLP 2018 can dig into this more, but my assumption is that you'd have to contact the authors.

**Comment by mjpost** [comment]
Yes, it seems these authors forgot to submit it. If you contact them, you could let them know they could send it to me for uploading by creating an Issue here.

You can see what attachments look like on other papers, e.g., https://aclweb.org/anthology/papers/D/D18/D18-1512/

**Comment by yucc2018** [comment]
can anyone supply NAACL 2019 papers?

**Comment by mjpost** [comment]
NAACL 2019 papers will be available in the Anthology on on June 2, 2019.

**Comment by aryamccarthy** [comment]
The main page of the anthology is slightly wider than all other pages. Moving to one of those other pages means the top bar contents jump closer together.

**Comment by mbollmann** [comment]
> The main page of the anthology is slightly wider than all other pages. Moving to one of those other pages means the top bar contents jump closer together.

Yes, that's because the overview tables are so wide and should use all the space to prevent scrolling. I don't think there's a good way to address this without a complete overhaul of how we present all the conferences on the main page.

**Comment by jwtaki** [comment]
the searching does not work?

**Comment by akoehn** [comment]
@jwtaki are you in a country that block google by any chance?  The search functionality is provided by them so search does not work in China for example.

**Comment by allanj** [comment]
New suggestion: it would be better to (have an option to) divide the papers in each year into different areas. Right now, we have a lot of papers different from 10 years ago when we could still read all of them. 

I tried to read most of the papers but I have to select some of them. Thus, the area of interest is the key factor for me and other readers to read. Probably.

**Comment by akoehn** [comment]
@allanj the information is not stored in the meta data, so someone would have to classify all papers before any frontend logic could be written.

That is a lot of work and it is often not clear at all what categories should be used and one that is decided, which category a paper belongs to.  Look at the high rates of papers moved between areas after the _authors_ have selected a category.

**Comment by TobiasLee** [comment]
It would be better to support searching for papers by institute name.

**Comment by aryamccarthy** [comment]
Like categories, this isn't stored in the metadata. We'd have to either extract it from the PDFs (which is noisy) or do it manually (which is so large as to be infeasible).

And on a personal note, I worry about the consequences of providing search by institution. Our community has become large enough that we can't read every paper, and search by institution may lead to a rich-get-richer bias in which papers get read. That's not a problem in itself; the problem is the other side of the scale: high-quality papers will be overlooked because of the institution they come from. (I believe that authors write papers, not institutions. The same could happen with author search, which is supported, but it's harder to make that systemic and entrenched.)

**Comment by TobiasLee** [comment]
@aryamccarthy Thanks for your considerable reply.

**Comment by Evpok** [comment]
> @TobiasLee Like categories, this isn't stored in the metadata. We'd have to either extract it from the PDFs (which is noisy) or do it manually (which is so large as to be infeasible).

Consequences aside, [GROBID](https://github.com/kermitt2/grobid) is quite efficient at extracting such metadata from pdfs. [HAL](https://hal.inria.fr/) uses it to prefill metadata for new deposits.

**Comment by mayhewsw** [comment]
This is a tiny tiny thing, but it always bothers me that the yellow banner on top ("You're viewing the latest version...") is too close to the header. 

If you have the time and patience for such trivia, you can fix this by adding padding to the parent div (container).

```html
<div class="container" style="padding-top:15px">
  <aside class="alert alert-warning text-center py-1 mt-n3 mt-md-n4 mt-xl-n5" role="alert">You're viewing the latest version of the ACL Anthology.
    <a class="btn btn-warning mx-2" href="https://github.com/acl-org/acl-anthology/issues/170">Give feedback</a>
  </aside>
</div>
```

**Comment by mjpost** [comment]
Fixed live and I agree it looks better. Want to submit a PR? `hugo/_default/baseof.html` I believe.

**Comment by mbollmann** [comment]
> Fixed live and I agree it looks better. Want to submit a PR? `hugo/_default/baseof.html` I believe.

Please use Bootstrap classes instead of custom style attributes, though. (Or play around with removing the explicit negative margin classes of the banner.)

**Comment by iyuge2** [comment]
awesome, thanks

**Comment by enueno** [comment]
I'd like to know the exact date were published the Proceedings of the 2018 Conference on Empirical Methods in Natural Language Processing. Also the actual papers do not display the CC-BY licencse, are they bound but that CC-BY licencse as stated in the webpages? the paper I am interested on, states that 
c 2018 Association for Computational Linguistics, what seems like an 'All rights reserved' llicense. Many thanks. If this is not the forum to find out this information, where / who could I contact? Perhaps it would be useful to find an email on the website for queries. Many thanks. Best wishes, Eva

**Comment by akoehn** [comment]
@enueno The conferences currently do not have an exact published date in the anthology. You could use the starting date of the conference as publication dates, which is, according to https://emnlp2018.org/, 2018-11-02.

Yes, ACL holds the copyrights for the papers. However, ACL licenses the papers under a CC-by license so you can use it when attributing.

**Comment by faustusdotbe** [comment]
Hi @mbollmann ,

The latest update (*SEM proceedings 2019) has some mistakes in properly parsing people's names, eg https://aclweb.org/anthology/volumes/proceedings-of-the-eighth-joint-conference-on-lexical-and-computational-semantics-sem-2019/ has "University of Amsterdam Ekaterina Shutova", and other mistakes.

Thank you for your work!

**Comment by omidrohanian** [comment]
> On the author page, e.g. https://www.aclweb.org/anthology/people/y/yan-song/, we can only see the count for the top 5 most frequent venues (it used to be possible to view the count for all venues):
> 
> ![image](https://user-images.githubusercontent.com/15331/55843178-c7f44680-5aeb-11e9-8290-f84a52ee35af.png)

I totally agree with this. I prefer to see all the venues which is more informative.

**Comment by rueycheng** [comment]
Abstracts can be quite useful for readers to quickly skim through the entire proceedings when made accessible via mouseover texts (e.g. [ACL '18 accepted paper page](https://acl2018.org/programme/papers/)). Do you plan to provide this sort of functionality? Not all bibtex records contain this info though.

**Comment by mjpost** [comment]
@rueycheng where do you envision this information being added? On the conference index pages? Post an example link if you like.

**Comment by rueycheng** [comment]
@mjpost Thanks for asking. Yeah I think having this info in conference index pages would be ideal, e.g. <https://aclweb.org/anthology/events/acl-2018/>. Or in a separate page ("view abstracts") that can be accessed from conference index in case this might slow down page loading.

**Comment by saamc** [comment]
Search result list contains results as direct links to PDF documents (sometimes only those) as well as links to document pages. Document pages contain metadata and are much more useful, and getting to a document page from a PDF is cumbersome.
https://www.aclweb.org/anthology/papers/P/P15/P15-2087/ (document metadata page)
vs
https://www.aclweb.org/anthology/P15-2087 (PDF document)
Also the URL on the document metadata page does not refer to itself but the document.
If a search finds a hit inside a document and not in the metadata could the corresponding document page be returned instead of the document itself?
It is obviously easier to use the bibliographic information provided on the document metadata page. E.g. Zotero('s ACL importer) doesn't manage to resolve from the PDF back to the metadata page and after a direct document import tries to fetch bibliographic information from a generic catalog that often doesn't contain the full authoritative information available on ACL web.

**Comment by mjpost** [comment]
It's a bit of an undocumented feature, but you can get to the document page easily by appending a `/`:

- <https://www.aclweb.org/anthology/P15-2087> → takes you to the PDF
- <https://www.aclweb.org/anthology/P15-2087/> → takes you to the paper page

**Comment by saamc** [comment]
Thanks, that's nifty indeed!

**Comment by mbollmann** [comment]
> Abstracts can be quite useful for readers to quickly skim through the entire proceedings when made accessible via mouseover texts (e.g. [ACL '18 accepted paper page](https://acl2018.org/programme/papers/)). Do you plan to provide this sort of functionality?

@rueycheng  That is a great idea IMO!  I can look into that.

> If a search finds a hit inside a document and not in the metadata could the corresponding document page be returned instead of the document itself?

@saamc  We are using Google Custom Search at the moment, so we do not have the ability to implement this level of customization. The only option available is to restrict search to a specific subset of documents, which is why there's the "Paper Metadata" tab which explicitly searches in and returns only the document pages, not the PDFs. However, excluding the PDFs means no search results based on the PDFs either, so that's really only a compromise right now.

**Comment by danielhers** [comment]
![Skærmbillede fra 2019-06-14 10-10-52](https://user-images.githubusercontent.com/263363/59494109-c26d0100-8e8c-11e9-96ec-30150873456e.png)
The drop-down for the sort criterion does not expand vertically to fit the contents on my screen.

**Comment by Aspie96** [comment]
The URL of the front page of volumes is crazy long.
It should be possible to link it (not the PDF directly, but the metadata also) without including such a long URL.

I understand having the title in the URL for SEO purposes, but a short URL should be provided.

**Comment by mjpost** [comment]
@Aspie96 Can you link to an example of what you mean?

**Comment by Aspie96** [comment]
@mjpost https://aclweb.org/anthology/volumes/sem-2012-the-first-joint-conference-on-lexical-and-computational-semantics-volume-1-proceedings-of-the-main-conference-and-the-shared-task-and-volume-2-proceedings-of-the-sixth-international-workshop-on-semantic-evaluation-semeval-2012/

The URL of the PDF is:
https://www.aclweb.org/anthology/S12-1

For a paper:
https://aclweb.org/anthology/papers/S/S12/S12-1000/
(the title is the same, but the content not).

The PDF:
https://www.aclweb.org/anthology/S12-1000

It would be much easier to reference the first link if it was https://aclweb.org/anthology/papers/S/S12/S12-1/.

**Comment by reyha** [comment]
All the earlier links like https://www.aclweb.org/anthology/W16-0404 are breaking.

**Comment by akoehn** [comment]
> All the earlier links like https://www.aclweb.org/anthology/W16-0404 are breaking.

@reyha The URLs work just fine for me.  Could you open a new issue detailing how exactly it breaks for you? (maybe try to reproduce it on a new browser / computer first)

**Comment by nschneid** [comment]
What about allowing people to list a personal website on their author pages? Would be another thing to maintain, but I guess it could be done with a Google form and a script.

**Comment by mjpost** [comment]
@nschneid I think maintaining dynamic data like this would be a huge pain. The form (like what I'm doing for attachments) would help but even that is time-consuming. Maybe if we get the Anthology tied to ACL Portal accounts we could do this.

**Comment by knmnyn** [comment]
+1 for getting such user information from the Portal or softconf instead.

Cheers,

Min

--
Min-Yen KAN (Dr) :: Associate Professor :: National University of Singapore
:: NUS School of Computing, AS6 05-12, 13 Computing Drive
Singapore 117417 :: +65 6516 1885(DID) :: +65 6779 4580 (Fax) ::
kanmy@comp.nus.edu.sg (E) :: www.comp.nus.edu.sg/~kanmy (W)

**Comment by akoehn** [comment]
@nschneid, @mjpost there is already an ID system we could support instead: orcid.  Let's manage such dynamic data in one place "industry"-wide instead of adding new identifiers.  One can add links on their orcid side and that way changes only need to be managed in one place.

**Comment by nschneid** [comment]
ORCID sounds good to me. But would that require curation? Can it be linked from the ORCID end?

I just checked my START profile and, oddly enough, it doesn't have a website field.

**Comment by mjpost** [comment]
I don't think website is relevant to START. But the Portal (under a redesign planned by @desilinguist) could one day incorporate dynamic user information (website, START ID, anthology author page, ORCID).

**Comment by knmnyn** [comment]
Hi Arne, all:

I talked with Marti and the past TACL and CL leadership about ORCIDs and
they were generally aboard on this matter, but I understood that MIT Press
was going under some big overhaul that included a discussion about
canonical Author IDs.  You may want to check with both CL / TACL heads
(Hwee Tou Ng and Ani Nenkova) on this.  I think they were waiting for the
Anthology to make a move first, as they had even more work on the
operations side than I did (then in 2017-8).

Cheers,

Min

--
Min-Yen KAN (Dr) :: Associate Professor :: National University of Singapore
:: NUS School of Computing, AS6 05-12, 13 Computing Drive
Singapore 117417 :: +65 6516 1885(DID) :: +65 6779 4580 (Fax) ::
kanmy@comp.nus.edu.sg (E) :: www.comp.nus.edu.sg/~kanmy (W)

**Comment by aryamccarthy** [comment]
The Volume page Spoken Language Translation [here](https://aclweb.org/anthology/volumes/spoken-language-translation/) does not exist. Same with the [Tutorials volume](https://aclweb.org/anthology/volumes/tutorials/). I couldn't concoct any other volume names to test out, but both pages appear on Google.

**Comment by sudacn** [comment]
good！

**Comment by Runze-huang** [comment]
I can't search for any papers？

**Comment by akoehn** [comment]
runze writes:

> I can't search for any papers？

It uses google custom search.  If google doesn’t work for you, the
search won’t as well.  If it is something different, a more detailed
report would be helpful.

**Comment by LuckyJLin** [comment]
I can't read this PDF https://www.aclweb.org/anthology/W19-3604
There is nothing in the page.

**Comment by akoehn** [comment]
> I can't read this PDF https://www.aclweb.org/anthology/W19-3604
> There is nothing in the page.

There is a PDF, but the PDF is empty.  Maybe an error in the ingestion
process?

**Comment by LuckyJLin** [comment]
> I can't read this PDF https://www.aclweb.org/anthology/W19-3604
>  There is nothing in the page.
> There is a PDF, but the PDF is empty.  Maybe an error in the ingestion
> process?

I open this PDF from previous page https://www.aclweb.org/anthology/papers/W/W19/W19-3604/
but I still get an empty PDF.

**Comment by mjpost** [comment]
That is the PDF that the Widening NLP workshop gave us. There are many other empty (W19-3601 W19-3604 W19-3607 W19-3618 W19-3629 W19-3644 W19-3648) and improperly-formatted papers.

**Comment by alphadl** [comment]
Thanks, I am trying to find the WMT19's papers in the set, but I can not find them.

**Comment by mjpost** [comment]
They have not been ingested yet. Please see statmt.org/wmt19 where they are available.

**Comment by jihunchoi** [comment]
It seems that ACL 2019 tutorial abstracts have *ACL 2017* publication information on their footer (see https://www.aclweb.org/anthology/P19-4#page=11 for example); is it intended?

**Comment by mjpost** [comment]
@jihunchoi Forgotten rsync, fixed, thank you!

**Comment by sedimentation-fault** [comment]
I want to commend you and thank you for putting so much thought into a naming system of papers and their related information, like bibtexs. This is by no means self-evident - and in fact I have seen it only at ACL. It is incredible, but you are the only people on the planet who name related resources with the same basename! This puts you light years ahead of your time! 

Let me explain:

Suppose I look at the aclweb.org site and say to myself: "WOW! What an incredible treasure. I would love to download it all and have it in my local paper library for my reading pleasure!". Well, that's easy. I remember looking at it 10 years ago - or even further back in the past. It has always been easy to "get them all". But that's only part of the story. Having PDFs named like "pennington-etal-2014-glove.pdf" does not help at all - you must rename them to some naming scheme amenable to searching, e.g.:

Venue Volume Issue Year DOI Authors Title

For example, for the above paper:

Proceedings 2014 Conference on Empirical Methods in Natural Language Processing EMNLP 2014 [doi 10.3115%2Fv1%2FD14-1162] Pennington, Jeffrey; Socher, Richard; Manning, Christopher -- Glove - Global Vectors for Word Representation.pdf

Notice that you can reconstruct a basic bibtex from the above name, knowing that semicolons delimit author names, the tile comes after ' -- ', the DOI is the URL-encoded string XXX in '[doi XXX]', the year is the 4-digit string before the DOI part and "Venue" is before that.

For this to work, you need bibliographic information for each paper, say in the form of a .bib file. You have that - everybody has that. But what you have - and everybody else is still missing - is this:

The paper and its associated bibliographic information have the same basename! That is, if the above paper has a URL

https://www.aclweb.org/anthology/D14-1162

then I know that the paper is at

https://www.aclweb.org/anthology/D14-1162.pdf

and its associated bibtex at

https://www.aclweb.org/anthology/D14-1162.bib

I can get those two just by looking at the 'url={...}' lines of the 'cumulative' bibtex at

https://www.aclweb.org/anthology/anthology.bib.gz

and as soon as I have two files, one PDF and one BIB, with the same basename

D14-1162.pdf
D14-1162.bib

I *know* they are connected! 

It's so simple, but its impact is immense. Imagine you would have a PDF

D14-1162.pdf

but its bibtex would have a different basename, say

pennington-etal-2014-glove.bib

How on earth would you know they belong together? You would have to resort to web scraping: parse each proceedings HTML page and, for each PDF link on it, find the '.bib' HTML link that is visually 'nearest' to it. This is programming hell.

Having a local paper collection, with papers renamed as above, makes searching (an issue that has been the subject of quite a few postings above) a dream: just list your local papers and pipe the list to a text file. Now use that text file as a "poor man's index" using, say, grep. You can grep it with any regular expression you like 

grep -E 'your regexp' index.txt

If you rename your local papers as above, you will be amazed at what you can find by such a simple method!

So thank you for making local collections possible with such genial ideas like providing a cumulative .bib file and using consistent names across the whole site for both papers and their bibliographic information. Forget OAI-PMH, federated repository aggregators and all that! All a truly open access paper repository needs is those two simple things!

**Comment by lucy3** [comment]
It's possible my brain is pudding right now, but is there a way to navigate to EMNLP Findings papers from the homepage of the ACL anthology? I see they're posted here: https://www.aclweb.org/anthology/volumes/2020.findings-emnlp/, but ctrl-F for "Findings" on the main page or the EMNLP page doesn't lead to any results.

**Comment by mjpost** [comment]
Hi @lucy3—it's not currently linked from the front page, but will be soon.

**Comment by Pranav-Goel** [comment]
A paper I have not authored was wrongly assigned to my ACL Anthology page because I have the exact same name as the first author on that paper. How do I get it removed from my profile?

**Comment by mbollmann** [comment]
@Pranav-Goel Please open a new issue for that, and make sure to include the Anthology ID(s) of the paper(s) in question. We can disambiguate authors in the metadata then. If you have an academic website and/or an ORCID ID, feel free to include a link to it too, as it might help us with the disambiguation process.

**Comment by AmyOlex** [comment]
When search results come up for a keyword search it would be helpful to see the data of publication and the list of authors.  Some of the PDFs don't have any dates in the footer.  Also, would there be a way to subscribe to a certain search result and get email updates when new papers are posted that match?

**Comment by BramVanroy** [comment]
The form that is linked to in the side bar "The Anthology can archive your poster or presentation! Please submit them in PDF format by filling out this form." is not accessible anymore.

**Comment by AGalassi** [comment]
Not sure if this is the right place where to ask these things, please redirect me if this is the wrong place:
- Would it be possible to show the ORCID identifier of the author on the author page?
- Are ACL anthology publications indexed in services such as Scopus and Web of Science?

**Comment by akoehn** [comment]
ORCID: We do not have orcid data for authors, so currently not. See eg.g https://github.com/acl-org/acl-anthology/pull/1179 for WIP.

**Comment by zixiu-alex-wu** [comment]
Hi, first of all, thank you for your work on the ACL Anthology website --- it's amazing!

My question is about the (lack of) Scopus indexing of my paper (https://aclanthology.org/2021.clpsych-1.22/), accepted to the Seventh Workshop on Computational Linguistics and Clinical Psychology, co-located with NAACL 2021.

Right now, the paper is NOT indexed on Scopus, and upon further digging, I have found that neither the workshop itself (2021 occurrence) nor NAACL 2021 is Scopus-indexed, despite the fact that previous proceedings of both are indexed. I was wondering if you could help to make sure that my paper and the proceedings of the workshop and NAACL 2021 are indexed on Scopus? Without the indexing, my paper will not count towards my PhD degree.

I have tried to contact anthology@aclweb.org about this multiple times, but I have not heard from them.

Thank you!

Best regards,

Zixiu Wu

**Comment by mbollmann** [comment]
@zixiu-alex-wu Have you tried asking Scopus about this? I'm not aware of anything we do on our side related to indexing papers in other databases, and would be surprised if we had any control about this.

**Comment by mjpost** [comment]
We have been manually submitting proceedings here and there, as we find motivated volunteers to do so. I’m in the process of working with a volunteer to be more systematic about this, but it’s currently something we are not handling well. It is on our 2022 roadmap, however!

**Comment by zixiu-alex-wu** [comment]
> @zixiu-alex-wu Have you tried asking Scopus about this? I'm not aware of anything we do on our side related to indexing papers in other databases, and would be surprised if we had any control about this.

Hi Marcel, thank you for your response! I have actually asked Scopus already, and they have an investigation underway, so I thought I'd ask the ACL anthology people about this as well. 

In fact, the workshop's organiser referred me to the chairs of NAACL 2021, who in turn referred me to "the ACL Anthology folks", because, as he put it, "they are maybe the only ones who would know the most recent year of NAACL is not indexed yet in Scopus".

Once Scopus informs me of the results of their investigation, I will put an update here.

Thank you again for your response!

**Comment by zixiu-alex-wu** [comment]
> We have been manually submitting proceedings here and there, as we find motivated volunteers to do so. I’m in the process of working with a volunteer to be more systematic about this, but it’s currently something we are not handling well. It is on our 2022 roadmap, however!

Hi Matt,

Thank you for your response!

So, if I am not mistaken, the proceedings of ACL conferences such as NAACL, as well as the proceedings of the co-locating workshops, have mostly been submitted to Scopus manually, which has resulted in the indexing of the proceedings of the conferences in previous years.

In that case, I was wondering if you could perhaps arrange for the proceedings of the workshop in question (https://aclanthology.org/volumes/2021.clpsych-1/) as well as the proceedings of NAACL 2021 to be submitted to Scopus, so that both of them as well as my workshop paper (https://aclanthology.org/2021.clpsych-1.22/) would be indexed?

Thank you!

Best regards,

Zixiu Wu

**Comment by AGalassi** [comment]
> We have been manually submitting proceedings here and there, as we find motivated volunteers to do so. I’m in the process of working with a volunteer to be more systematic about this, but it’s currently something we are not handling well. It is on our 2022 roadmap, however!

Hi, does this apply only for the main ACL conferences or also for non-ACL events, such as COLING?

**Comment by mjpost** [comment]
The ACL can only assume responsibility for ACL events, unless some arrangement is made.

**Comment by AGalassi** [comment]
> The ACL can only assume responsibility for ACL events, unless some arrangement is made.

Thank you! Since these other events are present in the ACL anthology as well I was not sure if they were managed independently or not.

**Comment by mbollmann** [comment]
Closing this as the “new” website is now over 4 years old. Feedback is still welcome, but can go into more specific issues.
